integrations/viewing-data-in-an-integration.md

```diff
@@ -1,5 +1,24 @@
 # Viewing Data in an Integration
 
-We're hard at work on a new feature that will show you a preview of the data in your connected integrations, but it's not quite ready yet.
+## Base Data Artifacts
+These are the data artifacts that exist in the integration that wasn't loaded there via the Aqueduct `.load` function.
```

Comment:
Grammatical nit -- should be weren't rather than wasn't. 

Rather than saying via the Aqueduct API, why don't we say that these are data artifacts that weren't created by an Aqueduct workflow?